Face detection minor/docker compose

nakuldesai21 opened the pull request:
Please review the DockerFile,docker-compose, and README file as well.

nakuldesai21 committed:
added Docker file

nakuldesai21 committed:
docker-compose file created

nakuldesai21 committed:
Update README.md

update the README.md file by pushing Docker and docker-compose files.

NiniiGit commented on Dockerfile:
```diff
@@ -0,0 +1,7 @@
+COPY . /app/
+RUN pip install -r requirements.txt
+EXPOSE 8000
+CMD [ "python3","manage.py","runserver","0.0.0.0:8000" ]
```
any particular reason to use CMD command? How do you decide which one you will use when between `CMD vs ENTRYPOINT` ==> add a note for it in README.md

NiniiGit commented on README.md:
```diff
@@ -1,2 +1,60 @@
+
+----------------------------------------------
+## Deployment using Docker container
+## pre-requisite
```
you need to mention that this is for ubuntu based system installation

NiniiGit commented on README.md:
```diff
@@ -1,2 +1,60 @@
+  - cd Face-Detection-Minor1
+  - docker image build -t <image_name> DockerFile
+  - docker images ls
+  - docker run -p 8000:8000 <image_name> 
```
add a line above to mention what this command do?

NiniiGit commented on docker-compose.yml:
```diff
@@ -0,0 +1,17 @@
+version: '3.3'
+
+# services:
+#  django:
+#   build: .
+#   ports:
+#     - "8000:8000"
+ # command: python3 manage.py runserver
```
remove this one if this is not needed

NiniiGit commented on docker-compose.yml:
```diff
@@ -0,0 +1,17 @@
+    - "8000:8000"
+  volumes:
+    - .:/app
+  command: python3 manage.py runserver 0.0.0.0:8000
```
why are we passing this command here - isn't it part of image already ?

nakuldesai21 committed:
Update README.md

I am updating the README.md file to use the Application by applying Kubernetes replicaset and Kubernetes deployment object.

nakuldesai21 committed:
updated docker-compose file

nakuldesai21 committed:
Merge branch 'Face-Detection-Minor/docker-compose' of https://github.com/nakuldesai21/Face-Detection-Minor1 into Face-Detection-Minor/docker-compose

nakuldesai21 committed:
added ENTERPOINT

nakuldesai21 committed:
created new github action file for compose branch

nakuldesai21 committed:
updating action file

nakuldesai21 committed:
updated readme.md

nakuldesai21 committed:
deleted readme.md file

nakuldesai21 committed:
added actions file to build and push docker Image in registery

nakuldesai21 committed:
updated readme.md file

nakuldesai21 committed:
Update docker-action.yml

added some comments
